fetch.bs

```diff
@@ -1192,38 +1192,45 @@ or "<code>worker</code>".
   <p><a>In parallel</a>, while true:
 
   <ol>
-   <li><p>Wait for <var>read</var> to be fulfilled or rejected.
-
    <li>
-    <p>If <var>read</var> is fulfilled with an object whose <code>done</code> property is false
-    and whose <code>value</code> property is a <code>Uint8Array</code> object, then run these steps:
+    <p>Run these substeps, but abort the substeps if fetch is <a for=fetch>terminated</a>:</p>
```

Comment:
As noted in https://github.com/whatwg/infra/issues/143 I'm not sure there's any reason to use "substeps" over "steps" here and elsewhere.